ansible/roles/test/files/ptftests/py3/copp_tests.py

```diff
@@ -44,7 +44,7 @@ class ControlPlaneBaseTest(BaseTest):
     MAX_PORTS = 128
     PPS_LIMIT = 600
     PPS_LIMIT_MIN = PPS_LIMIT * 0.9
-    PPS_LIMIT_MAX = PPS_LIMIT * 1.3
+    PPS_LIMIT_MAX = PPS_LIMIT * 1.4
```

Comment:
Please limit the platform and give a comment why need to tune the value, is there any accuracy issue?

Reply:
Done. The 8111 hw statistical policer's policing is not very accurate.